packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -245,6 +252,30 @@ export const Datepicker: ComponentWithAs<'div', DatepickerProps> &
     }
   };
 
+  const onInputFocus = ev => {
+    if (!props.allowManualInput) {
+      setOpenState(OpenState.Opening);
+      ev.preventDefault();
+    }
+  };
+
+  const onInputBlur = () => {
+    if (props.autoCorrectManualInput && !!error) {
+      setFormattedDate(valueFormatter(selectedDate));
+      if (selectedDate) {
```

Comment:
Planning to switch to `useReducer` of handling states (formattedDate, selectedDate and error)

Reply:
After further discussion with shift, this will not be implemented currently.